### DISCUSSION TITLE
Having trouble with a daily PDF report

### DISCUSSION THREAD

**Discussion by cbg-dev** [jsvine/pdfplumber#604] [Ask for help with specific PDFs]
Hello,

I am trying to extract the names from this PDF file and hopefully get it in a format where I can just output the names into an excel spreadsheet like below in a single cell.

[testpdf.pdf](https://github.com/jsvine/pdfplumber/files/8073177/testpdf.pdf)

![image](https://user-images.githubusercontent.com/76846662/154115186-0f10b87b-8520-4682-bfb4-03bc85810a6f.png)

i'm having trouble trying to figure out how to isolate for just the names and not the time/size on the right.

the image below is what i am aiming for

![image](https://user-images.githubusercontent.com/76846662/154115307-a472fcd1-0409-4123-928c-ace2d604e69a.png)

**Comment by jsfenfen** [comment]
You might look at the table extraction example. https://github.com/jsvine/pdfplumber/blob/stable/examples/notebooks/extract-table-nics.ipynb

**Comment by jsvine** [comment]
In addition to @jsfenfen's suggestion, using `Page.crop(bbox)` may help: https://github.com/jsvine/pdfplumber#the-pdfplumberpage-class

... that is: Crop the page to just the left half, and then run the text extraction on the cropped page instance.
